src/test/java/teammates/test/cases/browsertests/InstructorCourseDetailsPageUiTest.java

```diff
@@ -239,14 +241,13 @@ private InstructorCourseDetailsPage getCourseDetailsPage() {
         return loginAdminToPage(detailsPageUrl, InstructorCourseDetailsPage.class);
     }
 
-    private boolean didStudentReceiveReminder(String courseName, String courseId, String studentEmail,
-                                              String studentPassword)
+    private boolean didStudentReceiveReminder(String courseName, String courseId, String studentEmail)
```

Comment:
It's technically fine, right? Just not following the style guide but still fits a boolean.